beer/cli/subcommands/hmm/mkphoneloop.py

```diff
@@ -65,7 +68,7 @@ def main(args, logger):
     logger.debug('compiling the graph...')
     cgraph = graph.compile()
 
-    categorical = priors[args.weights_prior](len(start_pdf))
+    categorical = priors[args.weights_prior](len(start_pdf), len(start_pdf)/2)
```

Comment:
shouldn't the second argument here be `args.concentration` unless `args.concentration` is -1

Reply:
Yep, you are right. It should be fixed now.